cmd/zpool/zpool_main.c

```diff
@@ -6556,6 +6565,48 @@ status_callback(zpool_handle_t *zhp, void *data)
 	return (0);
 }
 
+#ifdef _UZFS
```

Comment:
I don't think we need this ifdef, the code will work properly for in-kernel ZFS, too.